### PR TITLE
Use `/dev/urandom` in CI

### PR DESCRIPTION
**Pull request by ArturGajowy**
## Overview
<sup>_What this PR does, and why it's needed_</sup>

### JIRA ticket:
<sup>_Create it if there isn't one already._</sup>

### Notes
<sup>_Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else._</sup>

### Please make sure that this PR:
- [x] is at most 200 lines of code (excluding tests),
- [x] meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards),
- [x] includes tests for all added features,
- [x] has a reviewer assigned,
- [x] has [all commits signed](https://rchain.atlassian.net/wiki/spaces/DOC/pages/498630673/How+to+sign+commits+to+rchain+rchain).

### [Bors](https://bors.tech/) cheat-sheet:

- `bors r+` runs integration tests and merges the PR (if it's approved),
- `bors try` runs integration tests for the PR,
- `bors delegate+` enables non-maintainer PR authors to run the above.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -77,6 +77,8 @@ pipeline:
     group: parallel
     <<: *sbttestenv
     commands:
+      - rm /dev/random
+      - mknod /dev/random c 1 9
```
Comment (odeac) [inline .drone.yml]: should't this be /dev/urandom?

File: .drone.yml [hunk 2]
```diff
@@ -87,6 +89,8 @@ pipeline:
     <<: *sbttestenv
     commands:
       - cp -r . ../step.run-casper-tests && cd ../step.run-casper-tests
+      - rm /dev/random
+      - mknod /dev/random c 1 9
```
Comment (odeac) [inline .drone.yml]: same here

**Comment by ArturGajowy** [comment]
This was not expecting a review yet, and was just a failed experiment :)

File: .drone.yml (hunk 1 above)
Comment (ArturGajowy) [reply .drone.yml]: Nope, we're replacing /dev/random with the same thing that /dev/urandom normally resolves to

**Pull request closed without merging**